packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -844,44 +815,38 @@ export function didNotFindHydratableSuspenseInstance(
   }
 }
 
-export function mountEventComponent(
-  eventComponentInstance: ReactDOMEventComponentInstance,
-): void {
-  if (enableFlareAPI) {
-    const rootContainerInstance = ((eventComponentInstance.rootInstance: any): Container);
-    const doc = rootContainerInstance.ownerDocument;
-    const documentBody = doc.body || doc;
-    const responder = eventComponentInstance.responder;
-    const {
-      rootEventTypes,
-      targetEventTypes,
-    } = ((responder: any): ReactDOMEventResponder);
-    if (targetEventTypes !== undefined) {
-      listenToEventResponderEventTypes(targetEventTypes, documentBody);
-    }
-    if (rootEventTypes !== undefined) {
-      addRootEventTypesForComponentInstance(
-        eventComponentInstance,
-        rootEventTypes,
-      );
-      listenToEventResponderEventTypes(rootEventTypes, documentBody);
-    }
-    mountEventResponder(eventComponentInstance);
-  }
-}
-
-export function updateEventComponent(
-  eventComponentInstance: ReactDOMEventComponentInstance,
-): void {
-  // NO-OP, why might use this in the future
-}
-
-export function unmountEventComponent(
-  eventComponentInstance: ReactDOMEventComponentInstance,
+export function mountResponderInstance(
+  responder: ReactDOMEventResponder,
+  responderInstance: ReactDOMEventResponderInstance,
+  props: Object,
+  state: Object,
```

Comment:
Can we call this `stateNode` or something to distinguish it from React state? The key distinction to me is that it's mutable and lives outside React's update cycle

Reply:
`responderState`?